cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -871,6 +870,64 @@ public CloudServiceBroker getServiceBroker(String name, boolean required) {
         return fetchListWithAuxiliaryContent(this::getServiceResources, this::zipWithAuxiliaryServiceOfferingContent);
     }
 
+    @Override
+    public LastOperation getServiceLastOperation(String serviceName) {
+        Resource<UnionServiceInstanceEntity> resource = getServiceInstanceResourceByName(serviceName).block();
+        if (resource == null) {
+            throw new CloudOperationException(HttpStatus.NOT_FOUND, "Not Found", "Service instance " + serviceName + " not found.");
+        }
+        return resource.getEntity()
+                       .getLastOperation();
```

Comment:
Why not include the `LastOperation` object in `CloudServiceInstance`, remove this method and just use `getServiceInstance` instead?